__tests__/commands/Build/appName.test.ts

```diff
@@ -0,0 +1,66 @@
+import { program, CommanderStatic } from 'commander';
+import {
+  jestCommon,
+  createTempDir,
+  createTemplate,
+  initProject,
+  linkDirCustom,
+  removeTempDir
+} from '../../test-helpers';
+import buildCommand from '../../../src/commands/Build/buildCommand';
+
+const PROJECT_NAME = 'test-project';
+const ORIGINAL_CWD = linkDirCustom();
+const TEMP_DIR = ORIGINAL_CWD + '/buildappNameTemp';
+const APP_NAME = 'test-app';
+const OPTIONS = ['node', 'build', '--app-name', APP_NAME];
+const { afterAll, beforeAll, describe, expect, test } = jestCommon;
+
+describe('appName', () => {
+  let mainProgram: CommanderStatic;
+
+  beforeAll(async () => {
+    createTempDir(TEMP_DIR);
+
+    await initProject(TEMP_DIR, PROJECT_NAME);
+    await createTemplate(TEMP_DIR, PROJECT_NAME);
+
+    mainProgram = buildCommand(program);
+    process.argv = OPTIONS;
+
+    await mainProgram.parseAsync(process.argv);
+  });
+
+  afterAll(() => {
+    removeTempDir(TEMP_DIR);
+  });
+
+  test('Should be "test-app" when assign to "test-app"', async () => {
+    expect(mainProgram.opts().appName).toBe('test-app');
+  });
+});
+
+describe('appName', () => {
```

Comment:
I updated code here:
https://github.com/kintone-labs/kintone-cli/pull/119/commits/d7001edf02e724a0cd562ae32b8ecc6b384cd4d7

Please have a check.
Thank you!

Reply:
You keep `describe('appName empty'` in addition to `describe('appName'` because you wanna reset the test env with beforeAll and afterAll again, right?
If so, I think it's ok to have two describe cases.

Reply:
And if going with the current implementation, it will be more understandable with 'empty appName' than 'appName empty' I think.

Reply:
Please check the comment above.